### PR TITLE
Cv user checklist

### PR DESCRIPTION
**Pull request by cvializ**

### CODE REVIEW

**Comment by cvializ** [comment]
@ac @ajkochanowicz

File: kifi-backend/modules/shoebox/angular/src/recos/recos.styl [hunk 1]
```diff
@@ -42,6 +42,9 @@ twoColMinWidth = appSidePad1 + leftColWidth + colGapWidth + rightColWidth + appS
   margin: 0 auto 50px
   max-width: rightColWidth + 20px
 
+  > * + *
```
Comment (adamjgrant) [inline kifi-backend/modules/shoebox/angular/src/recos/recos.styl:45]: That's quite the selector.

**Comment by adamjgrant** [comment]
:+1:

File: kifi-backend/modules/shoebox/angular/src/recos/recos.styl (hunk 1 above)
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/recos/recos.styl:45]: I was hoping you'd notice that. I've found it useful in the past and I wanted to suggest it:
http://alistapart.com/article/axiomatic-css-and-lobotomized-owls

Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/recos/recos.styl:45]: I can totally get rid of it though, but I wanted to discuss it first : P

File: kifi-backend/modules/shoebox/angular/src/keep/keepCard.styl [hunk 2]
```diff
@@ -253,24 +253,25 @@ defaultBorderColorOverlayUsingBefore(borderWidth)  // slightly darkens an image'
...
-    &:first-child
-      margin-left: -(sidePad)
-
-    &:focus
-    &:hover
-      box-shadow: inset 0 0 0 1px #f6f6f6
-
-      &:active
-        background-color: #fcfcfc
+.kf-keep-keep-btn
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/keep/keepCard.styl:256]: If you're wondering about these changes (and the changes to recoLibraryCard), it's to make the Follow Library and Keep button and look the same regardless of whether they're inside or outside of a keep/library card

**Pull request merged**